packages/gatsby-theme-docs/src/components/top-menu.js

```diff
@@ -6,6 +6,7 @@ import { css } from '@emotion/core';
 import SpacingsInline from '@commercetools-uikit/spacings-inline';
 import SpacingsStack from '@commercetools-uikit/spacings-stack';
 import { designSystem } from '@commercetools-docs/ui-kit';
+import UserResearchBanner from './user-research-banner';
```

Comment:
I remember that we wanted to make this a generic place to post banners ("top-menu-banner") and not hardwire the specific current content into the code structure. 

I suggest to make the banner overridable by placing an empty "top-menu-banner-area" into the overrides folder which then imports and uses the actual user-research-banner component.

Reply:
Hmm, right. Sorry I had it different in mind 👍